wslib-classes/GUI/Extensions/extServer-makeView.sc

```diff
@@ -172,7 +172,7 @@
 		w.front;
 
 		ctlr = SimpleController(this)
-			.put(\serverRunning, {	if(this.serverRunning,running,stopped) })
+			.put(\this.serverRunning, {	if(this.serverRunning,running,stopped) })
```

Comment:
this doesn't look right.  its supposed to be a symbol called 'this.serverRunning' ?  should just be \serverRunning